tests/jsondb.test.js

```diff
@@ -126,6 +126,14 @@ Deno.test("methodDelete", () => {
     data: { ...json, laptops: json.laptops.slice(1) },
   });
 
+  jsonCopy = JSON.parse(JSON.stringify(json));
```

Comment:
Just `{ ...json }` instead?

Reply:
It can do only one-level copy though, here we need deep copy.